### PR TITLE
edb path length property added

### PR DESCRIPTION
**Pull request by svandenb-dev**
was missing feature

### CODE REVIEW

**Review by ring630 — APPROVED** [review]
LGTM

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/edb_core/edb_data/primitives_data.py [hunk 1]
```diff
@@ -709,6 +709,22 @@ def width(self, value):
             else:
                 self.primitive_object.SetWidth(value)
 
+    @property
+    def length(self):
+        """Return the path length.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/primitives_data.py:714]:
```suggestion
        """Path length.
```
A description for a property shouldn't include a verb.

File: pyaedt/edb_core/edb_data/primitives_data.py [hunk 2]
```diff
@@ -709,6 +709,22 @@ def width(self, value):
...
+
+        Returns
+        -------
+        float
+            Path length in meter.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/primitives_data.py:719]:
```suggestion
            Path length in meters.
```

**Pull request merged**